### ISSUE TITLE
scans.export() w/ multiple chapters.

### ISSUE THREAD

**Issue by hdsiles** [tenable/pyTenable#70]
**scans.export chapters does not work with more than one value.**
When attempting to run a scans.export() with more than a single list entry I always get back a blank report with only the headers.  I get this same result with the pdf and html formats defined. 

**To Reproduce**
Based on examples in documentation I should be able to run 
with open(dl_loc, 'wb') as reportobj:
    tio.scans.export(scan_id=1, fobj=reportobj, format='html', chapters=['vuln_by_host', 'compliance'])

If I limit above to a list with a single entry:  ['vuln_by_host'] OR ['compliance'] the results seem to work fine.

I do not see an error message of any kind.  The export function does create a file, but the information inside seems to be missing under the circumstances mentioned above.  I would have to run some level of debug to track the issue down farther. 

**Expected behavior**
I would expect one report type and then the other within the same html/pdf file, but it does not seem to work correctly if more than one entry in 'chapters' is set.

**System Information (please complete the following information):**
Running MacOS 10.14.3

Let me know if there are more details to provide.

**Comment by hdsiles** [comment]
After getting a chance to debug I think I found where the problem was.
According to the tenable api docs:
https://cloud.tenable.com/api#/resources/scans/export-request
chapters in regards to scan exports `expecting a semi-colon delimited string`.

It looks like in the join for chapters in the scans.py file is using a comma instead.  In my own test I changed it to a semi-colon and that seems to have resolved the issue.

I will see about submitting a fix from a fork.
